docs/css/tc-header.css

```
@@ -102,6 +102,20 @@ body.mobile-menu #site-header #mobile-menu-toggle {
     white-space: nowrap;
 }
 
+#site-header .menu-item .badge {
+    color: white;
+    font-size: 10px;
+    padding: 2px 6px;
+    background-color: #0FD5C6; // somehow $topaz is too dark for me.
```

Do we keep this comment here?
```suggestion
    background-color: #0FD5C6;
```

---

No need 😅